### PR TITLE
Updated the Nix cache information in the README

### PR DESCRIPTION
**Pull request by koslambrou**
Copied from plutus.

Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] Commit sequence broadly makes sense
    - [ ] Key commits have useful messages
    - [ ] Formatting, PNG optimization, etc. are updated
- PR
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description
    - [ ] Reference the ADR in the PR and reference the PR in the ADR (if revelant)
    - [ ] Reviewer requested

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -192,8 +192,9 @@ To set up the cache:
 . On non-NixOS, edit `/etc/nix/nix.conf` and add the following lines:
 +
 ----
-substituters        = https://hydra.iohk.io https://iohk.cachix.org https://cache.nixos.org/
-trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= iohk.cachix.org-1:DpRUyj7h7V830dp/i6Nti+NEO2/nhblbov/8MW7Rqoo= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
+substituters = https://hydra.iohk.io https://cache.nixos.org/
```
Comment (zeme-iohk) [inline README.adoc:195]: We want to add `https://cache.iog.io` here too

**Review by zeme-iohk — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 1 above)
Comment (koslambrou) [reply README.adoc:195]: In that case, it's missing in the https://github.com/input-output-hk/plutus/blob/master/CONTRIBUTING.adoc#installing-and-setting-up-nix repo. I'll add it there as well.

Comment (koslambrou) [reply README.adoc:195]: Should we remove `https://cache.nixos.org`? Or do we need to add it in the NixOS config?

Comment (zeme-iohk) [reply README.adoc]: That's the default cache. I don't know if it's used implicitly by NixOS if no cache is defined. I'd keep it, it doesn't hurt.

**Review by zeme-iohk — APPROVED** [review]

**Pull request merged**